i3-wk-switch.py

```diff
@@ -10,10 +10,10 @@
 import logging
 import sys
 from pprint import pformat
+import i3ipc
```

Comment:
Move this 3rd party import after time, with a newline in between (where the `import i3` used to be).